src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, Arial, sans-serif;
+}
+
+.form {
+  position: relative;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
```

Comment:
why do you need this?

Reply:
Form in position: relative, for my img to be in position: absolute.
Or you about "top, right,..."?